keyboards/ms_sculpt/config.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Jean Bernard (@jn-bernard)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+##pragma once
+
+#define RGB_DI_PIN A1
+#define RGBLED_NUM 16
+
+#define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```